### ISSUE TITLE
Azure Storage Queue Documentation incorrect

### ISSUE THREAD

**Issue by spartymants** [MicrosoftDocs/feedback#2842]
I've identified an issue with the Azure Storage Rest API documentation for the delete method. 

The delete message documentation does not appear to be correct
[Delete Message](https://docs.microsoft.com/en-us/rest/api/storageservices/delete-message2#emulated-storage-service-uri)

The document states that the following uri must be used
https://myaccount.queue.core.windows.net/myqueue/messages/messageid?popreceipt=string-value

There is no mention of messageid path attribute in the doc other than it be included in the uri.

if the **/messageid** (the literal or the actual message id from the azure storage queue item) is included in the path it results in an error 

![image](https://user-images.githubusercontent.com/11237066/84184317-09cca400-aa85-11ea-807c-0d38af858c6e.png)

```xml
<?xml version="1.0" encoding="utf-8"?><Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:a732d09d-4003-0018-8088-3eee34000000
Time:2020-06-09T18:08:29.5167204Z</Message>
<QueryParameterName>popreceipt</QueryParameterName>
<QueryParameterValue>AgAAAAMAAAAAAAAAYUMWAIg 1gE=</QueryParameterValue>
<Reason>Invalid pop receipt format</Reason>
</Error>
```
If the **/messageid** is removed then a 204 is returned as expected

![image](https://user-images.githubusercontent.com/11237066/84184180-d68a1500-aa84-11ea-8c47-76fe54ed7793.png)

Happy to help fixing the doc if you point me at the relevant location on github.

**Comment by guillene** [comment]
This seems to be a different bug, I get the same error when the pop receipt has a `+` in the middle. As you can see you submitted `AgAAAAMAAAAAAAAAYUMWAIg+1gE=` but you got the error that `AgAAAAMAAAAAAAAAYUMWAIg 1gE=` format is invalid

**Comment by yehudaQ** [comment]
@guillene 
The same problem happens to me too.
Did you find a solution for the bug?

**Comment by guillene** [comment]
Yes, you need to encode the URL before you send it. Some languages have libraries for that, in my case, I did a simple string replace every time I find a "+" to a "%2b" and works like a charm 😄

**Comment by yehudaQ** [comment]
@guillene 
Thank you so much!! 
As you said, works like a charm!

**Comment by Nzglobal** [comment]
Frustrating that the official documentation has this issue, and that it hasn't been fixed in over 2 years.

**Comment by lucasmaj** [comment]
OMG I just spend 1h chasing it. The documentation really should be more obvious.

https://myaccount.queue.core.windows.net/myqueue/messages/messageid?popreceipt=string-value

The instruction does talk about replacing myaccount  and popreceipt, but not messageid. Not it is obvious, but took to long.

**Comment by panfilenok-epam** [comment]
Just tell me where the page source is. I will create the PR
